website/static/templates.json

```diff
@@ -6,7 +6,8 @@
     "author": "Davide Antelmo",
     "source": "https://github.com/Azure-Samples/azure-search-openai-demo-java",
     "tags": [
-        "featured",
```

Comment:
this repo has also been reviewed by Java dev eng team: Bruno Borges and John Oliver. It's going to be presented at Ignite. You might consider to review tags and set it as featured.

Reply:
@dantelmomsft when we updated awesome-azd's ui last week, we also made some changes to our tags-- one change being that 'featured' is no longer a tag.

Since the template is housed in Azure-Samples, was reviewed by Java dev eng team, and is going to be presented at Ignite, you could consider changing the "community" tag to "msft".